plugins/bufbuild/connect-es/v0.13.0/buf.plugin.yaml

```diff
@@ -17,3 +17,4 @@ registry:
         version: ^0.13.0
 spdx_license_id: Apache-2.0
 license_url: https://github.com/bufbuild/connect-es/blob/v0.13.0/LICENSE
+deprecated: true
```

Comment:
It would be good to update `cmd/latest-plugins/main.go` method `latestNonDeprecatedPlugins` to use this to filter out deprecated plugins.